timeboost-types/src/bundle.rs

```diff
@@ -295,6 +297,38 @@ impl Committable for SignedPriorityBundle {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
+pub struct DkgBundle {
+    committee_id: CommitteeId,
+    transcript: Vec<u8>, // TODO: replace with VESS transcript (dealing)
```

Comment:
- you could directly use `struct VessCiphertext` that's already availabile in `vess.rs` 
- I'd add code doc on `DkgBundle` that this is a dealing from a DKG party, a dealing contains the Feldman commitment and encrypted ciphertext for _all_ other parties.